app/assets/stylesheets/forever_style_guide/modules/_nav-dropdowns-help.scss

```diff
@@ -0,0 +1,24 @@
+$supporting-text-icon-offset: 33px;
+.nav-dropdown-help {
+  .icon-offset {
+    display: block;
+    padding-left: $supporting-text-icon-offset;
+  }
+}
+.dropdown-menu-help {
+  width: 100%;
```

Comment:
You are completely right. I had copied over the code from the already existing dropdown-menu-account class wanting it to look exactly the same. Turns out this is not being used in the dropdown-menu-account either.